lib/driver.js

```diff
@@ -416,7 +416,9 @@ class AndroidDriver extends BaseDriver {
     }
     // some cleanup we want to do regardless, in case we are shutting down
     // mid-startup
-    await this.adb.stopLogcat();
+    if (!this.opts.skipLogcatCapture) {
```

Comment:
I think it's safe not to check this, since `stopLogcat` simply returns silently if no logs collections has been started

Reply:
Okay, right. Will simplify here
https://github.com/appium/appium-adb/blob/84d736c6b64a66721b634e69494f8ffaa47cba81/lib/tools/adb-commands.js#L1119-L1128